src/lifecycle.js

```diff
@@ -51,90 +52,141 @@ function parseEvent (e) {
   };
 }
 
-function triggerAttributeChanged (target, component, data) {
-  var callback;
-  var type;
-  var name = data.name;
-  var newValue = data.newValue;
-  var oldValue = data.oldValue;
-  var newValueIsString = typeof newValue === 'string';
-  var oldValueIsString = typeof oldValue === 'string';
-
-  if (!oldValueIsString && newValueIsString) {
-    type = 'created';
-  } else if (oldValueIsString && newValueIsString) {
-    type = 'updated';
-  } else if (oldValueIsString && !newValueIsString) {
-    type = 'removed';
-  }
+/**
+* Camel-cases the specified string.
+*
+* @param {String} str The string to camel-case.
+*
+* @returns {String}
+*/
+function camelCase (str) {
```

Comment:
Possibly. It's only used here, was just keeping it private until it needed to be public? Will think about it.